src/index.js

```diff
@@ -37,12 +37,21 @@ class ServerlessPlugin {
       );
     }
 
+    this.v3Utils = v3Utils;
     this.loadEnv(this.getEnvironment(options));
   }
 
-  log(...args) {
+  log(msg) {
     if (this.config.logging) {
-      this.serverless.cli.log(...args);
+      if (this.v3Utils) {
+        if (msg.includes('WARNING')) {
+          this.v3Utils.log.warning(msg);
+        } else {
+          this.v3Utils.log.notice(msg);
+        }
```

Comment:
Doing a string search is probably fine for this PR, but I think it's best for the caller to decide between `.warning()` and `.notice()`.

Was there a similar `this.serverless.cli.warn()` in v2?

Reply:
No, there wasn't a `cli.warn` previously and only with this new logging mechanism we've introduced proper logging levels. 

I fully agree that called should be responsible for deciding the log level of the message, but on the other hand I wanted to keep the changes as small as possible. What do you think makes more sense from the perspective of the plugin here?

Reply:
I'm happy to do a refactor after merging this PR in. Thanks!